Stripe/Resources/Images/stp_card_unionpay.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we support SVG images, so you may have to convert these

Reply:
Right, looks like SVG doesnt work because we're not using an asset catalog that gets compiled.